src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -792,37 +792,20 @@ Public Class BuildDevDivInsertionFiles
                         Continue For
                     End If
 
-                    If dependencies.ContainsKey(partFileName) Then
+                    If IsLanguageServiceRegistrationFile(partFileName) Then
                         Continue For
                     End If
 
-                    Dim relativeOutputDir As String
-                    If IsLanguageServiceRegistrationFile(partFileName) Then
-                        relativeOutputDir = Path.Combine(GetExternalApiDirectory(), "LanguageServiceRegistration", vsixName)
-                    Else
-                        relativeOutputDir = GetExternalApiDirectory()
+                    If dependencies.ContainsKey(partFileName) Then
+                        Continue For
                     End If
 
-                    Dim relativeOutputFilePath = Path.Combine(relativeOutputDir, partFileName)
+                    Dim relativeOutputFilePath = Path.Combine(GetExternalApiDirectory(), partFileName)
 
                     If processedFiles.Add(relativeOutputFilePath) Then
-                        If IsLanguageServiceRegistrationFile(partFileName) Then
-                            Dim absoluteOutputFilePath = GetAbsolutePathInOutputDirectory(relativeOutputFilePath)
-                            WriteVsixPartToFile(vsixPart, absoluteOutputFilePath)
-
-                            ' We want to rewrite a few of these things from our standard vsix-installable forms
-                            Select Case Path.GetExtension(absoluteOutputFilePath)
-                                Case ".pkgdef"
-                                    RewritePkgDef(absoluteOutputFilePath)
-
-                                Case ".vsixmanifest"
-                                    RewriteVsixManifest(absoluteOutputFilePath)
-                            End Select
-                        Else
-                            ' paths are relative to input directory:
-                            filesToInsert.Add(New NugetFileInfo(partFileName))
-                            AddXmlDocumentationFile(filesToInsert, partFileName)
-                        End If
+                        ' paths are relative to input directory:
+                        filesToInsert.Add(New NugetFileInfo(partFileName))
```

Comment:
@tmat: I guess I phrased my question poorly. Whatever it's doing, does it still need to be done? Wondering if there's more to delete. :-)